contracts/BaseStrategy.sol

```diff
@@ -176,6 +178,11 @@ abstract contract BaseStrategy {
         keeper = _keeper;
     }
 
+    function setRewards(address _rewards) external {
+        require(msg.sender == strategist || msg.sender == governance(), "!authorized");
```

Comment:
Seems like something governance shouldn't need rights to? They can always change `strategist` to themselves, then change this
```suggestion
        require(msg.sender == strategist, "!authorized");
```